back_end/RayTracer/Controllers/RenderController.cs

```diff
@@ -65,15 +65,17 @@ public HttpResponseMessage Post([FromBody] JsonObject requestJSON)
 

```

Comment:
In this file, just uncommenting the code that will return the response, as an image, with a specific file name, chosen by the user.